base/markdown/Julia/interp.jl

```diff
@@ -39,7 +39,7 @@ end
 
 toexpr(x) = x
 
-toexpr(xs::Vector{Any}) = Expr(:cell1d, map(toexpr, xs)...)
+toexpr(xs::Vector{Any}) = Expr(:vectany, map(toexpr, xs)...)
```

Comment:
Since the syntax that parses to `cell1d` is deprecated, maybe it makes sense just to leave this until we have an official new meaning for it?